site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+2:58:05AM: Delete started (2s ago)
+2:58:07AM: Deleting 
+	    | Target cluster 'https://10.92.0.1:443'
+	    | Changes
+	    | Namespace  Name                         Kind           Conds.  Age  Op      Op st.  Wait to  Rs  Ri
+	    | default    simple-app                   Deployment     2/2 t   26m  delete  -       delete   ok  -
+	    | ^          simple-app                   Endpoints      -       26m  -       -       delete   ok  -
+	    | ^          simple-app                   Service        -       26m  delete  -       delete   ok  -
+	    | ^          simple-app-5894d79b7f        ReplicaSet     -       26m  -       -       delete   ok  -
+	    | ^          simple-app-5894d79b7f-wxjrd  Pod            4/4 t   26m  -       -       delete   ok  -
+	    | ^          simple-app-stnf4             EndpointSlice  -       26m  -       -       delete   ok  -
+	    | Op:      0 create, 2 delete, 0 update, 4 noop, 0 exists
+	    | Wait to: 0 reconcile, 6 delete, 0 noop
+	    | 9:28:05PM: ---- applying 6 changes [0/6 done] ----
+	    | 9:28:05PM: noop endpointslice/simple-app-stnf4 (discovery.k8s.io/v1) namespace: default
+	    | 9:28:05PM: noop replicaset/simple-app-5894d79b7f (apps/v1) namespace: default
+	    | 9:28:05PM: noop pod/simple-app-5894d79b7f-wxjrd (v1) namespace: default
+	    | 9:28:05PM: noop endpoints/simple-app (v1) namespace: default
+	    | 9:28:05PM: delete deployment/simple-app (apps/v1) namespace: default
+	    | 9:28:05PM: delete service/simple-app (v1) namespace: default
+	    | 9:28:05PM: ---- waiting on 6 changes [0/6 done] ----
+	    | 9:28:05PM: ongoing: delete pod/simple-app-5894d79b7f-wxjrd (v1) namespace: default
+	    | 9:28:05PM: ok: delete service/simple-app (v1) namespace: default
+	    | 9:28:05PM: ok: delete endpoints/simple-app (v1) namespace: default
+	    | 9:28:05PM: ongoing: delete replicaset/simple-app-5894d79b7f (apps/v1) namespace: default
+	    | 9:28:05PM: ok: delete deployment/simple-app (apps/v1) namespace: default
+	    | 9:28:05PM: ongoing: delete endpointslice/simple-app-stnf4 (discovery.k8s.io/v1) namespace: default
+	    | 9:28:05PM: ---- waiting on 3 changes [3/6 done] ----
+	    | 9:28:06PM: ok: delete replicaset/simple-app-5894d79b7f (apps/v1) namespace: default
+	    | 9:28:06PM: ok: delete endpointslice/simple-app-stnf4 (discovery.k8s.io/v1) namespace: default
+	    | 9:28:06PM: ongoing: delete pod/simple-app-5894d79b7f-wxjrd (v1) namespace: default
+	    | 9:28:06PM:  ^ Deleting
+	    | 9:28:06PM: ---- waiting on 1 changes [5/6 done] ----
+	    | 9:28:13PM: ok: delete pod/simple-app-5894d79b7f-wxjrd (v1) namespace: default
+	    | 9:28:13PM: ---- applying complete [6/6 done] ----
+	    | 9:28:13PM: ---- waiting complete [6/6 done] ----
+2:58:13AM: App 'simple-app' in namespace 'default' deleted 
+
+Succeeded
+```
+Now that we know how we can talk to our App CRs, let's see how taking a closer look at them improves the package consumption experience.
```

Comment:
```suggestion
Now that we know how we can talk to our App CRs, let's take a closer look at how they improve the package consumption experience.
```

Reply:
I was trying to get across that "observing App CRs improve the pacakge consumption experience"
The suggested change seems to suggest "App CRs improve the package consumption experience"

Maybe the sentence could be restructured though? 🤔

Reply:
(Lol I could literally change it to this ^, does that sound better in your opinion?)

Reply:
Yeah, this sounds much better. I got really confused while reading the existing sentence and it felt as if there was some typo.

Reply:
Alright! making this change